### PR TITLE
Concepts review

### LINKED ISSUE #70: Review concepts section
- fix and missing links (e.g. [link a las tasks] or issues with reference to guides)
- include a brief description about metadata
- remove docstrings from methods and add a mention to the Python Client API reference.
- include a short introduction paragraph after the main title: In this section, we will bla bla, then we can have two subsections Rubrix data model (it's called now Main Concepts) or just Data model, and then Methods. In general we should avoid having a H1 with no text before the first H2, even short explanations of the content of that page will greatly help the readers.
- do a quick proof reading (there's a typo with POS tagging)
- I will provide you with updated figures to improve readability.

Thanks, very good job in general👍

### PR DESCRIPTION
**Pull request by ignacioct**
Closes #70 

Things left:

- [x] New figures

### CODE REVIEW

**Comment by ignacioct** [comment]
New figures added

File: docs/getting_started/concepts.rst [hunk 1]
```diff
@@ -3,14 +3,18 @@
 Concepts
 ========
 
-Rubrix Basic Concepts
----------------------
+In this section, we will introduce Rubrix basic concepts and the methods of the Python library that implement them. They are the way in which us, the users, communicate with the service, so it's important to understand what's happening behind the curtain.
```
Comment (dvsrepo) [inline docs/getting_started/concepts.rst]:
```suggestion
In this section, we introduce the core concepts of Rubrix. These concepts are important for understanding how to interact with the tool and its core Python client.
```

File: docs/getting_started/concepts.rst [hunk 2]
```diff
@@ -3,14 +3,18 @@
 Concepts
 ========
 
-Rubrix Basic Concepts
----------------------
+In this section, we will introduce Rubrix basic concepts and the methods of the Python library that implement them. They are the way in which us, the users, communicate with the service, so it's important to understand what's happening behind the curtain. 
+
+We have two main subsections: Rubrix data model (explaining those main concepts Rubrix is built around) and Python client API methods (which implement those concepts and data structures).
```
Comment (dvsrepo) [inline docs/getting_started/concepts.rst]:
```suggestion
We have two main sections: Rubrix data model and Python client API methods.
```

**Review by dvsrepo — APPROVED** [review]

File: docs/getting_started/concepts.rst (hunk 2 above)
Comment (ignacioct) [reply docs/getting_started/concepts.rst]: me fui por las ramas jajjaja

**Pull request merged**